luigi/parameter.py

```diff
@@ -1007,6 +1007,8 @@ def parse(self, x):
         :param str x: the value to parse.
         :return: the parsed value.
         """
+        if isinstance(x, list):
+            x = json.dumps(x)
```

Comment:
It would be nice if we have some docs explaining this. Core luigi should avoid having hard to understand code.